style.css

```diff
@@ -89,23 +95,87 @@ a:hover {
     color: var(---black);
 }
 
+/* hero section */
+
+.hero {
+    position: relative;
+    width: 100vw;
+    height: 82vh;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: start;
+    color: var(--white);
+    background-image: url(./assets/hero-image_large.jpeg);
```

Comment:
It seems you're only using this image but in this commit you have a total of 3 images. Do we need all of them?

Reply:
deleted the web-unsplash image and used the small hero image in media query.